src/das-employer-favourites.ApplicationServices/Commands/GetApprenticeshipFavouritesRequest.cs

```diff
@@ -0,0 +1,10 @@
+using DfE.EmployerFavourites.ApplicationServices.Domain;
+using MediatR;
+
+namespace DfE.EmployerFavourites.ApplicationServices.Commands
+{
+    public class GetApprenticeshipFavouritesRequest : IRequest<ApprenticeshipFavourites>
```

Comment:
I'd be tempted to separate the Commands and Queries into different folders just for some separation.